optimizely/client/client.go

```diff
@@ -133,6 +133,35 @@ func (o *OptimizelyClient) GetEnabledFeatures(userContext entities.UserContext)
 	return enabledFeatures, nil
 }
 
+func (o *OptimizelyClient) Track(userContext entities.UserContext, eventKey string, eventTags map[string]interface{}) error {
+	if !o.isValid {
+		errorMessage := "Optimizely instance is not valid. Failing GetEnabledFeatures."
+		err := errors.New(errorMessage)
+		logger.Error(errorMessage, nil)
```

Comment:
err is shadowed , logger.Error takes nil ? 
something like this could be better:

err = errors.New("Optimizely instance is not valid. Failing GetEnabledFeatures.")
logger.Error("",err) or logger.Error(err.String(), err) // kind of redundant 
return err